src/component/carousel/Carousel.jsx

```diff
@@ -7,8 +7,12 @@ const Carousel = () => {
   const [activeIndex, setActiveIndex] = useState(0);
   const [carouselList, setCarouselList] = useState([]);
   useEffect(async () => {
-    const res = await axios.get("/api/carousels");
-    setCarouselList(res.data.carousels);
+    try {
+      const res = await axios.get("/api/carousels");
+      setCarouselList(res.data.carousels);
+    } catch (err) {
+      console.log(err);
```

Comment:
Instead of console you can show this error to the toast or and alert box.

You can use bootstrap alertbox too😁.

Reply:
I will implement it after completing the required functionalities, thanks though